README.md

```diff
@@ -1,84 +1,66 @@
+- Runs tests and builds benchmarks
+- Support for Haskell build tools, including [Haddock](https://hackage.haskell.org/package/haddock), [GHCJS](https://github.com/ghcjs/ghcjs) & [HLint](https://hackage.haskell.org/package/hlint)
+- doctest runner integration using either [doctest](https://hackage.haskell.org/package/doctest) or [cabal-docspec](https://github.com/phadej/cabal-extras/blob/master/cabal-docspec/MANUAL.md).
 
-### Quick-start instructions
+Usage
+---
 
-* Step 1: Clone and install this project in/from any directory
+See `haskell-ci --help` for details.
 
-    ```bash
-    $ git clone https://github.com/haskell-CI/haskell-ci.git
-    $ cd haskell-ci
-    $ cabal new-install haskell-ci:exe:haskell-ci
-    ```
+1. Install `haskell-ci`
 
-  or
+```bash
+cabal install haskell-ci
+```
 
-    ```bash
-    cabal new-install haskell-ci
-    ```
+2. Add a `tested-with` line to your .cabal file (e.g. `tested-with: GHC == 8.10.4 || == 9.0.1`)
 
-    *Note:* currently (2019-02-12) released version of `haskell-ci` installs
-    an executable named `make-travis-yml`
+3. Create a configuration file. A good starting point for this is the `dump-config` command eg
 
-* Step 2: Change directories to your project:
+``` shell
+haskell-ci dump-config > haskell-ci.config
+```
 
-    ```bash
-    $ cd path/to/your-project
-    ```
+4. Run the `github` or `travis` commands as appropriate to your workflow. eg
 
-* Step 3: Edit your project's `*.cabal` file to add a `Tested-With` line, such as this one:
+``` shell
+haskell-ci github '--config=haskell-ci.config' 'cabal.project' --output .github/workflows/haskell-ci.yml
+```
 
-    ```bash
-    $ cat your-project.cabal
-    ...
-    Tested-With: GHC ==8.6.3 || ==8.4.4 || ==8.2.2
-    ...
-    ```
-    
-    Add as many or as few GHC versions to test as you want.
+``` shell
+haskell-ci travis '--config=haskell-ci.config' 'cabal.project' --output .travis.yml
+```
 
-* Step 4: Generate a Travis file for your project:
+5. Push your project adding the files created, which should trigger the appropriate CI for your setup.
 
-    ```bash
-    $ # You run the following command from your project's directory, even
-    $ # though it references the script from the `haskell-ci` project
-    $ haskell-ci your-project.cabal --output .travis.yml
-    ```
+GitHub actions example:
 
-    Note: If you have multiple local Cabal projects that you wish to build together
-    using a `cabal.project` file, pass that file to haskell-ci instead:
     ```bash
-    $ haskell-ci cabal.project --output .travis.yml
+    $ git checkout master            # Check out `master`
+    $ git pull --ff-only             # Get the latest version of `master`
+    $ git checkout -b new_action     # Create a `new_action` branch
+    $ git add .github/workflows/haskell-ci.yml
+    $ git commit -m "New action"
+    $ git push -u origin new_action  # Push your branch upstream
     ```
```

Comment:
This code block (and the next one) is malformatted:

![image](https://user-images.githubusercontent.com/3664523/142269164-9a36e240-02e7-4265-bc46-8e7d3f0899ea.png)

Either indent the line above, so the code block is part of the numbered list, or un-indent the code block.